app/backend/Extensions/KeyVaultConfigurationBuilderExtensions.cs

```diff
@@ -6,7 +6,7 @@ internal static class KeyVaultConfigurationBuilderExtensions
 {
     internal static IConfigurationBuilder ConfigureAzureKeyVault(this IConfigurationBuilder builder)
     {
-        var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT");
+        var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT") ?? "https://kv-laoqvch7yt7iq.vault.azure.net/";
```

Comment:
We shouldn't fallback to hardcoded values here.
```suggestion
        var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT");
```

Reply:
OH,, THis is for local development. I forget to revert this change, thanks for catching